qpc/report/merge.py

```diff
@@ -120,34 +97,69 @@ def _get_report_ids(self):
         return not_found, report_ids, job_not_found, report_not_found
 
     def _validate_create_json(self, files):
+        """Validate the set of files to be merged.
+
+        :param files: list(str) of the files to be merged
+        """
+        # pylint: disable=too-many-branches,too-many-statements
         print(_(messages.REPORT_VALIDATE_JSON % files))
         all_sources = []
+        report_version = None
+        report_type = None
         report_id = None
         for file in files:
-            error = True
             if os.path.isfile(file):
+                details_report = None
                 with open(file) as lint_f:
                     try:
-                        json_data = json.load(lint_f)
+                        details_report = json.load(lint_f)
                     except json_exception_class:
                         print(_(messages.REPORT_JSON_DIR_FILE_FAILED % file))
                         continue
-                    sources = json_data.get('sources')
+
+                    # validate report type
+                    report_type = details_report.get('report_type', None)
+                    if not report_type:
+                        print(_(messages.REPORT_MISSING_REPORT_VERSION % file))
```

Comment:
It seems like we want an error message about `report_type here`, not `report_version`.

Reply:
Fixed check to be report_version